src/Color.cpp

```diff
@@ -43,6 +43,27 @@ Rgb& Rgb::operator=( Hsv hsv ) {
     return *this;
 }
 
+Rgb Rgb::operator+( Rgb in ){
+    uint16_t red = this->r + in.r;
+    uint8_t red_ = (uint8_t)(red<255) ? red : 255;
```

Comment:
- The project uses `_name` instead of `name_`. 
- Please, do not use C-style casts, use `static_cast` if required (actually not required here)
- The combination of the ternary operator without parenthesis and cast in one command is a little confusing. Either add parenthesis or omit the cast (as it should not be required here)